plugins/api-docs/src/components/OpenApiDefinitionWidget/OpenApiDefinition.tsx

```diff
@@ -89,23 +75,57 @@ const useStyles = makeStyles(theme => ({
           .parameter__in`]: {
         color: theme.palette.text.disabled,
       },
-      [`& .opblock-description-wrapper p,
-          .opblock-description-wrapper li,
-          .opblock-external-docs-wrapper p,
-          .opblock-title_normal p,
-          .response-control-media-type__accept-message,
-          .opblock .opblock-section-header>label,
-          .scheme-container .schemes>label,
-          .info .base-url,
-          .model`]: {
-        color: theme.palette.text.hint,
+      [`& table.model, 
+          .parameter__type,
+          .model.model-title,
+          .model-title,
+          .model span,
+          .model .brace-open,
+          .model .brace-close,
+          .model .property.primitive,
+          .model .renderedMarkdown,
+          .model .description,
+          .errors small`]: {
+        color: theme.palette.text.secondary,
       },
       [`& .parameter__name.required:after`]: {
         color: theme.palette.warning.dark,
       },
-      [`& .prop-type`]: {
+      [`& table.model, 
+          table.model .model,
+          .opblock-external-docs-wrapper`]: {
+        fontSize: theme.typography.fontSize,
+      },
+      [`& table.headers td`]: {
+        color: theme.palette.text.primary,
+        fontWeight: theme.typography.fontWeightRegular,
+      },
+      [`& .model-hint`]: {
+        color: theme.palette.text.hint,
+        backgroundColor: theme.palette.background.paper,
+      },
+      [`& .opblock-summary-method, 
+          .info a`]: {
+        fontFamily: theme.typography.fontFamily,
+      },
+      [`& .info, .opblock, .tab`]: {
+        [`& li, p`]: {
+          fontFamily: theme.typography.fontFamily,
+          color: theme.palette.text.primary,
+        },
+      },
+      [`& a`]: {
         color: theme.palette.primary.main,
       },
+      [`& .renderedMarkdown code`]: {
+        color: theme.palette.secondary.light,
+      },
+      [`& .property-row td:first-child`]: {
+        color: theme.palette.warning.dark,
```

Comment:
To double check, these are for example this right?

![image](https://user-images.githubusercontent.com/4984472/144138955-aafe4b68-16a4-44d0-97de-82e6f6ba09c4.png)

I feel that maybe we should leave this one as a more standard black/white non-colored text, because particularly the warning I think is a big extra dark in our default theme and I kinda worry it won't read that well. Not too fond of the colored text in this case either tbh.

Reply:
Ok. I'll move this one back to b/w. It's indeed more readable.

Reply:
![bwParam](https://user-images.githubusercontent.com/49567430/144235971-b5b566c3-6ca7-4e05-9c84-05600e1eabf9.png)